### PR TITLE
Fix configuration bugs

### PR DESCRIPTION
**Pull request by kmshin1397**
Two errors I ran into when spinning up Fritz locally are addressed here:
- The Kowalski token was being written to the configs encoded as binary (with `!!binary`) and that didn't agree with SP.
- The `homepage_grid` parameter was missing for the default app configs, leading to an error when trying to build the homepage from its template file. 

The rest of the changes are just from black, I think - not sure why they weren't changed before.

### CODE REVIEW

**Comment by kmshin1397** [comment]
Hm looks like CI actually doesn't have the issue I was having locally - investigating

File: fritz [hunk 1]
```diff
@@ -110,7 +110,7 @@ def check_config(cfg="fritz.defaults.yaml", yes=False, **kwargs):
             config["kowalski"]["server"]["JWT_EXP_DELTA_SECONDS"]
         ),
     )
-    config["skyportal"]["app"]["kowalski"]["token"] = kowalski_token
+    config["skyportal"]["app"]["kowalski"]["token"] = kowalski_token.decode("utf-8")
```
Comment (dmitryduev) [inline fritz:113]: Ah, I've finally realized what's going on: please revert this change and instead bump the required version for `pyjwt>=2.0` in fritz/.requirements/dev.txt. You probably have 1.7.x installed locally -- they made this change in 2.0. The nice thing is that the CI actually caught this!

File: fritz.defaults.yaml [hunk 2]
```diff
@@ -112,6 +112,40 @@ skyportal:
         icon: Info
         url: /about
 
+  homepage_grid:
```
Comment (dmitryduev) [inline fritz.defaults.yaml:115]: Nice catch! I wonder if we have it in prod...

File: fritz.defaults.yaml [hunk 3]
```diff
@@ -143,11 +177,11 @@ skyportal:
     WeatherWidget:
       resizeable: false
       layouts:
-        xlg: [ 0, 10, 4, 1 ]
-        lg: [ 0, 10, 4, 1 ]
-        md: [ 0, 10, 4, 1 ]
-        sm: [ 0, 10, 4, 1 ]
-        xs: [ 0, 12, 4, 1 ]
+        xlg: [0, 10, 4, 1]
```
Comment (dmitryduev) [inline fritz.defaults.yaml:180]: @acrellin was also reporting weird linting behavior. Might be worth looking into why this is happening.

**Review by dmitryduev — CHANGES_REQUESTED** [review]
👍🏻

File: fritz (hunk 1 above)
Comment (kmshin1397) [reply fritz]: Yup I just came to the same conclusion. Reverted and requirements have been bumped

File: fritz.defaults.yaml (hunk 3 above)
Comment (kmshin1397) [reply fritz.defaults.yaml:180]: Yea I ran `./fritz lint` to make sure and everything passed

File: fritz.defaults.yaml (hunk 2 above)
Comment (dmitryduev) [reply fritz.defaults.yaml:115]: We do, just checked.

**Review by dmitryduev — APPROVED** [review]
LGTM! Will merge once green.

**Pull request merged**